.github/workflows/cypress.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     services:
       collabora:
-        image: collabora/code
+        image: ghcr.io/juliushaertl/nextcloud-dev-code:latest
```

Comment:
does it stay in your namespace or will it hop over eventually?

Reply:
As mentioned in the pr description ideally this would be a official Collabora image on ghcr but for now this is a mirror with nightly updates